camel-route-support/src/test/java/io/kaoto/backend/api/service/step/parser/camelroute/CamelRouteStepParserServiceTest.java

```diff
@@ -62,4 +66,38 @@ void deepParse() throws IOException {
         assertThat(yaml).isEqualToNormalizingNewlines(route);
         assertTrue(camelRouteDeploymentGeneratorService.supportedCustomResources().isEmpty());
     }
+
+
+    @ParameterizedTest
+    @ValueSource(strings = {"route.yaml", "route2.yaml", "route3.yaml"})
```

Comment:
What do you think to give a more precise name to the files to pinpoint the part which is specifically tested?

Reply:
They were the examples that were given, but you are right, I should add more info